tests/urlpatterns/test_resolvers.py

```diff
@@ -13,3 +14,15 @@ class RoutePatternTests(SimpleTestCase):
 
     def test_str(self):
         self.assertEqual(str(RoutePattern(_('translated/'))), 'translated/')
+
+
+class ResolverCacheTests(SimpleTestCase):
+
+    @override_settings(ROOT_URLCONF='urlpatterns.path_urls')
+    def test_resolver_defaults_to_root_urlconf(self):
+        """Passing the ROOT_URLCONF explicitly is equivalent to passing no
+        argument, so the resolver should be the same cached object
+        """
+        # the `root_urlconf_changed` test signal will properly isolate this cache for us
+        get_resolver() is get_resolver('urlpatterns.path_urls')
+        get_resolver() is not get_resolver('urlpatterns.other_path_urls')
```

Comment:
Use `self.asserIs` and `self.assertIsNot` as these boolean expressions are noop.